Specs/Scene/PostProcessStageCompositeSpec.js

```diff
@@ -181,6 +181,15 @@ defineSuite([
         }
 
         expect(s).toRender([0, 0, 0, 255]);
+        // Dummy Stage
+        var bgColor = 51; // Choose a factor of 255 to make sure there aren't rounding issues
+        s.postProcessStages.add(new PostProcessStageComposite({
+            stages : [new PostProcessStage({
```

Comment:
Why does it have to be a `PostProcessStageComposite` wrapping a `PostProcessStage`. Could it just be a `PostProcessStage`?

Reply:
Honestly, I just copied the code below it and changed it. I was being lazy. Ill make the fix!

Reply:
Honestly, I just copied the code below it and changed it. I was being lazy. Ill make the fix!